add descheduler dedicated meeting

knelasevero opened the pull request:
Related to https://github.com/kubernetes-sigs/descheduler/issues/944

Adding a suggestion for the dedicated descheduler meeting that we discussed. Let's agree on a time and which meeting platform to use and links over this PR review.

Is it a requirement to use Zoom? What about the recordings, how does it usually work?

damemi commented on sigs.yaml:
```diff
@@ -2372,6 +2372,14 @@ sigs:
     archive_url: https://docs.google.com/document/d/13mwye7nvrmV11q9_Eg77z-1w3X7Q1GTbslpml4J7F3A/edit
     recordings_url: https://www.youtube.com/watch?v=PweKj6SU7UA&list=PL69nYSiGNLP2vwzcCOhxrL3JVBc-eaJWI
+  - description: descheduler biweekly meeting
+    day: Tuesday
+    time: "17:00"
```
could I suggest another time like `18:00` (2pm et) or `15:00` (11am et)? I'd like to be able to attend these

damemi replied:
Tuesdays work for me (misread this at first and thought thursday)

alculquicondor commented:
Yes, zoom is a requirement. @ahg-g or @Huang-Wei can help create the recurring meeting.

knelasevero commented:
Maybe aiming for tomorrow is too short notice. Next week then?

knelasevero commented:
> Yes, zoom is a requirement. @ahg-g or @Huang-Wei can help create the recurring meeting.

@alculquicondor @ahg-g @Huang-Wei Regarding uploading the recording, is it just creating a youtube playlist and getting that link to put it here?

And regarding creating and running those zoom meetings, I've noticed sometimes there are some problems with allowing people to share screen, can we make @damemi @ingvagabund @a7i and I (and maybe some other folks) being able to share screen by default?

Dentrax commented:
Thanks for PR! How can we get an `.ics` file to add Outlook calendar?

damemi commented:
> Maybe aiming for tomorrow is too short notice. Next week then?

next week puts this back on a recurring conflict for me :\ As long as we can get the zoom link+calendar invite set up then tomorrow should be fine, we can handle the playlist uploading afterward as long as we can record it

knelasevero commented:
> Thanks for PR! How can we get an `.ics` file to add Outlook calendar?

I think that if we add the scheduling google group to the event when we create it it would show up for anyone in that group. I am not sure, but I think this is what was done for the other meetings

knelasevero commented:
> > Maybe aiming for tomorrow is too short notice. Next week then?
> 
> next week puts this back on a recurring conflict for me :\ As long as we can get the zoom link+calendar invite set up then tomorrow should be fine, we can handle the playlist uploading afterward as long as we can record it

Cool, If we don't get the zoom link+calendar invite set up we cant find a way to meet anyways

JaneLiuL commented on sig-list.md:
```diff
@@ -49,7 +49,7 @@ When the need arises, a [new SIG can be created](sig-wg-lifecycle.md)
 |[Release](sig-release/README.md)|[release](https://github.com/kubernetes/kubernetes/labels/sig%2Frelease)|* [Stephen Augustus](https://github.com/justaugustus), Cisco<br>* [Sascha Grunert](https://github.com/saschagrunert), Red Hat<br>|* [Slack](https://kubernetes.slack.com/messages/sig-release)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-release)|* Regular SIG Mee ...
 |[Scalability](sig-scalability/README.md)|[scalability](https://github.com/kubernetes/kubernetes/labels/sig%2Fscalability)|* [Marcel Zieba](https://github.com/marseel), Google<br>* [Shyam Jeedigunta](https://github.com/shyamjvs), AWS<br>|* [Slack](https://kubernetes.slack.com/messages/sig-scalability)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-scale)|* Regular SIG  ...
-|[Scheduling](sig-scheduling/README.md)|[scheduling](https://github.com/kubernetes/kubernetes/labels/sig%2Fscheduling)|* [Wei Huang](https://github.com/Huang-Wei), Apple<br>* [Abdullah Gharaibeh](https://github.com/ahg-g), Google<br>|* [Slack](https://kubernetes.slack.com/messages/sig-scheduling)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-scheduling)|* biweekly mee ...
+|[Scheduling](sig-scheduling/README.md)|[scheduling](https://github.com/kubernetes/kubernetes/labels/sig%2Fscheduling)|* [Wei Huang](https://github.com/Huang-Wei), Apple<br>* [Abdullah Gharaibeh](https://github.com/ahg-g), Google<br>|* [Slack](https://kubernetes.slack.com/messages/sig-scheduling)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-scheduling)|* biweekly mee ...
```
17:00 UTC， this time i am sorry that i can not join, since i locate on UTC+8 :(

damemi replied:
I'm sorry @JaneLiuL  :\ we could use the current sig-scheduling APAC meeting to go over anything of note from the descheduler calls, would that work for you?

Otherwise I think it will be tough to find a good time for one meeting with that much of a distribution

knelasevero replied:
Do we have more people active in the descheduler project that are close to that timezone? @JaneLiuL another option is for someone to host another dedicated meeting in a convenient time for those in that area. @JaneLiuL would you like to host it?

knelasevero commented:
Hey all, timing was unfortunate... I'm sick today and won't be able to host. We call do it another day of the week if we want to do it fast, or Keep the schedule intact and do it in 2 weeks.

a7i commented:
> We call do it another day of the week if we want to do it fast, or Keep the schedule intact and do it in 2 weeks.

Sorry to hear that, get well soon! I'm :+1: for 2 weeks from today.

damemi commented:
> > We call do it another day of the week if we want to do it fast, or Keep the schedule intact and do it in 2 weeks.
> 
> Sorry to hear that, get well soon! I'm 👍 for 2 weeks from today.

+1, feel better @knelasevero

Huang-Wei commented:
@damemi @knelasevero you can create the meeting by yourself (there is no such privilege saying who can create a calendar IIUC), the guidance is here: https://www.kubernetes.dev/docs/comms/calendars/.

An important note is in the [Configure access permissions and sharing](https://support.google.com/calendar/answer/37082?hl=en) section: share the calendar with contributors@kubernetes.io and sig-scheduling-leads@kubernetes.io:

> Share it with full rights (“make changes and manage sharing”) to your SIG/WG lead mailing list and [contributors@kubernetes.io](mailto:contributors@kubernetes.io).

damemi commented:
Thanks Wei! @knelasevero do you want to go ahead and set up the meeting then? We can re-use the zoom link from the current sig-scheduling, just need the admin to record it

knelasevero commented:
For sure!

Will go over it tomorrow

knelasevero commented:
Ok, created a placeholder video on youtube just to create our playlist link: https://youtube.com/playlist?list=PLEafaZMk8MYYqoSncOkL8uutIcTsUQE_g

Here a doc for the meeting notes: https://docs.google.com/document/d/1Ang5shAFxuylO2RXMFGO7uxSe-IX3Y3PxoKIn1yPIEQ/edit#

And changed the link here to use the same zoom that is used for the other meetings.

Already created an event with the new calendar as owner and added relevant mailing lists as owner/viewer as per that guide.

I thing everything is set up

damemi commented:
Thanks @knelasevero, I got the invite through the sig-scheduling group so I think we're all set
/lgtm

Huang-Wei commented:
/approve

knelasevero commented:
/assign @nikhita

:)

palnabarun reviewed (approved):
/lgtm

jberkus commented:
/approve
